pom.xml

```diff
@@ -83,10 +83,8 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.17</version>
+        <version>2.22.0</version>
         <configuration>
-          <!-- Hiverunner needs this for some queries -->
-          <argLine>-Xmx1024m -XX:MaxPermSize=256m</argLine>
```

Comment:
Jacoco uses the argLine so if you override it you need to also add the value something else has set it to but since we don't actually appear to need this argLine at all (I think we just copied it over from HiveRunner which has tests requiring more resources) just removing it allows Jacoco to do its thing.